bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/keySelectionWindow.js

```diff
@@ -435,7 +435,40 @@ pimcore.object.classificationstore.keySelectionWindow = Class.create({
         this.store = new Ext.data.Store({
             remoteSort: true,
             proxy: proxy,
-            fields: readerFields
```

Comment:
The longer I think about it .... why not simply use a column renderer.

Example can be found here:
![image](https://user-images.githubusercontent.com/6142086/80688230-c6693800-8acb-11ea-8218-e02e2aa2a604.png)

Reply:
Now looks a lot better :smiley: